js/widgets/status.js

```diff
@@ -66,8 +68,8 @@ function StatusMatrix() {
 
         // One column per mouse/turtle
         let activeTurtles = 0;
-        for (let t = 0; t < this._logo.turtles.turtleList.length; t++) {
-            if (this._logo.turtles.turtleList[t].inTrash) {
+        for (let t = 0; t < logo.turtles.turtleList.length; t++) {
```

Comment:
Please consider using `for..of` loop here. Please do this across the file where it makes sense.